File: DCMTKSample/DCMTKSample/ViewController.m
//
//  ViewController.m
//  DCMTKSample
//
//  Created by Sean Ashton on 24/07/2015.
//  Copyright (c) 2015 Schimera Pty Ltd. All rights reserved.
//

#import "ViewController.h"
#include "DicomUtil.h"

@interface ViewController ()

@property (weak, nonatomic) IBOutlet UIImageView *imageView;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    [DicomUtil test];
    NSString *fullPath = [DicomUtil extractFirstFrame];
    NSLog(@"%@", fullPath);
    
    self.imageView.image = [UIImage imageWithContentsOfFile:fullPath];
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: DCMTKSampleSwift/DCMTKSampleSwift/ViewController.swift
//
//  ViewController.swift
//  DCMTKSampleSwift
//
//  Created by Sean Ashton on 29/03/2016.
//  Copyright © 2016 Schimera Pty Ltd. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    @IBOutlet weak var imageView: UIImageView!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        DicomUtil.test()
        let imagePath = DicomUtil.extractFirstFrame();       
        imageView.image = UIImage(contentsOfFile: imagePath!)
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

